src/components/TableContentSkeleton/TableContentSkeleton.test.tsx

```diff
@@ -0,0 +1,75 @@
+import { render } from '@testing-library/react'
```

Comment:
I think this component should located in `skeleton` folder too.

That change can be help us to improve the visibility of the exports

@AntonioFeregrino